extractedTranslations/en/translation.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
@cheloRydel this should be removed

Reply:
removing it in the next PR against this branch :)

Reply:
done